package.json

```diff
@@ -39,17 +46,19 @@
     "mcl-wasm": "^1.7.0"
   },
   "devDependencies": {
-    "@typechain/ethers-v6": "^0.5.1",
     "@jest/globals": "^29.7.0",
+    "@typechain/ethers-v6": "^0.5.1",
     "@types/jest": "^29.5.14",
     "@types/node": "^22.7.5",
     "dotenv": "^16.4.5",
     "esbuild": "^0.24.0",
     "eslint": "^9.17.0",
     "ganache": "^7.9.2",
     "jest": "^29.7.0",
+    "rimraf": "^6.0.1",
```

Comment:
Are rimraf or tsup actually used?

Reply:
They weren't used. I'll update the package.
I thought they'll be needed for publishing